src/app/admin/_components/Event/TagList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function TagList({ tags, isLoading, error }) {
+  return (
+    <div className="flex gap-2">
+      {!isLoading && error && <p>Error happened while fetching tags</p>}
+      {isLoading && !error && <p>Loading...</p>}
+      {!isLoading &&
+        !error &&
```

Comment:
🟠 i think conventions are fololwing:
- is data is pending, `isLoading: true`, and all other empty
- if data available, `isLoading: false` and error is absent
- if data not available `isLoading: false` and data is absent, error is present

so i guess those `&&` could be simplified

Reply:
simplified